invisible_cities/cities/beersheba_test.py

```diff
@@ -149,3 +149,15 @@ def test_beersheba_expandvar(deconvolution_config):
     conf['deconv_params']['psf_fname'] = '$ICDIR/database/test_data/PSF_dst_sum_collapsed.h5'
 
     beersheba(**conf)
+
+
+def test_beersheba_copy_kdst(deconvolution_config, ICDATADIR):
+    PATH_IN = os.path.join(ICDATADIR, "test_cdst_NEW_v1.2.0_bg.h5")
+    conf, PATH_OUT = deconvolution_config
+    conf["files_in"] = PATH_IN
+    conf["run_number"] = 8515
+
+    beersheba(**conf)
+
+    with tb.open_file(PATH_OUT) as output_file:
+        assert hasattr(output_file.root, "DST/Events")
```

Comment:
I would also check that the event numbers are those you expect.

Reply:
Added